tests/test_methods.py

```diff
@@ -302,7 +302,7 @@ def testUnsignedImages(self):
 
         # arrays differ by a maximum of 2
         errMsg1 = imagesDiffer(image0, image1)
-        self.assertIn("maximum absolute error: |0 - 2| = 2 at position (0, 1).", errMsg1)
+        self.assertRegex(errMsg1, r"maximum absolute error: \|0 - 2\| = 2 at position \(.*0.*,.*1\)" )
```

Comment:
@roceb now that I look at this patch I think the better fix is to patch the imagesDiffer function itself that generates the error message and put `int` calls into the f-strings.

@TallJimbo the issue here is that numpy changed the way repr works so you get an explicit `np.int64(1)` in the error message rather than `1`.

Reply:
I moved the fix to where message creation happens.